src/styles/main.scss

```diff
@@ -1,3 +1,39 @@
+@import "header";
+@import "card";
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.main {
+  max-width: 1200px;
+
+  &__card {
+    margin-bottom: 50px;
+  }
+
+  &--position {
+    margin: auto;
```

Comment:
it still block styles(modifier to block is block) - you cannot add margin here

Reply:
Also, bad modifier name

OK names:
- OR `--position--center`
- OR `--centered`

Reply:
created page__main element for page block